extensions/ql-vscode/src/model-editor/external-api-usage-queries.ts

```diff
@@ -32,7 +32,7 @@ type RunQueryOptions = {
   token: CancellationToken;
 };
 
-export async function prepareExternalApiQuery(
+export async function prepareModelEditorQueries(
```

Comment:
The queries we're referring to here are specifically the extraction/fetch queries, not all model editor queries right? If so, I'm not sure the new name is good enough.

Reply:
You're right that these are the extraction/fetch queries, but these are the only queries we currently have as "model editor" queries. They are [tagged as `modeleditor` in the CodeQL repository](https://github.com/github/codeql/blob/dbb4167f804e4b9dc5544a4a7bc19c3e4250ac2d/java/ql/src/utils/modeleditor/ApplicationModeEndpoints.ql#L6) and are also in a `modeleditor` directory. The method essentially recreates the file structure in that `modeleditor` directory, so I think this name does make sense.